keras_contrib/applications/densenet.py

```diff
@@ -604,36 +600,19 @@ def __create_fcn_dense_net(nb_classes, img_input, include_top, nb_dense_block=5,
 
     skip_list = skip_list[::-1]  # reverse the skip list
 
-    if K.image_dim_ordering() == 'th':
```

Comment:
Keras 2 deciphers the output shape of Deconvolution layers automatically​ now. Thus it is no longer to compute the output shape for which we needed a static batch size (none batch size did not work with Tensorflow before )